package.json

```diff
@@ -18,14 +18,19 @@
     "Other"
   ],
   "activationEvents": [
-    "onCommand:extension.open"
+    "onCommand:extension.open",
+    "onCommand:extension.addQuickNote"
   ],
   "main": "./extension.js",
   "contributes": {
     "commands": [
       {
         "command": "extension.open",
         "title": "Open Daily Notes"
+      },
+      {
+        "command": "extension.addQuickNote",
+        "title": "Add Quick Single Note"
```

Comment:
I really like to use "Daily Notes" here to make things consistent but I'm not sure how. Any thoughts?

Reply:
changes names to dailyNotes.insert